index.html

```diff
@@ -2191,60 +2164,18 @@ <h3>The <dfn>NDEFWriteOptions</dfn> dictionary</h3>
   <section data-dfn-for="NDEFScanOptions">
     <h3>The <dfn>NDEFScanOptions</dfn> dictionary</h3>
       <p>
-        To describe which messages an application is interested in, the
+        To pass an [= AbortSignal/aborted flag =], the
         <a>NDEFScanOptions</a> dictionary is used:
       </p>
       <pre class="idl">
         dictionary NDEFScanOptions {
-          USVString id;
-          USVString recordType;
-          USVString mediaType;
           AbortSignal? signal;
```

Comment:
Shall we now make `AbortSignal` mandatory as user would not pass a dictionary otherwise?
```suggestion
          AbortSignal signal;
```

Reply:
That is a good questions. We never know how things develop in the future, so I think it is fine that it is optional. Or do you want `scan({})` to fail?

Reply:
Formally it's still permissible to pass an empty dictionary as options, but if we want to stop that, yes, we could make it this way.  If we ever add back other options, we'll revert.

Reply:
Then the user code making use of the new API will fail outright in older browsers... seems bad

Reply:
OK, no big deal, could keep it nullable.

Reply:
Let's keep it optional then as I don't feel strongly about it.